File: Kotlin/Ejercicios/5.kt
/*
Escriba un programa que imprima los números del 1 al 4 en la misma línea. Escriba el programa utilizando los siguientes métodos:
a) Mediante una instrucción printf sin especificadores de conversión.
b) Mediante una instrucción printf con cuatro especificadores de conversión.
c) Mediante cuatro instrucciones printf.
*/

fun main() {
    println( "1 2 3 4")
    println( "%d %d %d %d".format(1, 2, 3, 4) )
    print( "1 " )
    print( "2 " )
    print( "3 " )
    print( "4 " )
}

File: Python/Ejercicios_capitulo_1/5.py
#Escriba un programa que imprima los números del 1 al 4 en la misma línea. Escriba el programa utilizando los siguientes métodos:
#a) Mediante una instrucción printf sin especificadores de conversión.
#b) Mediante una instrucción printf con cuatro especificadores de conversión.
#c) Mediante cuatro instrucciones printf.

print( "1 2 3 4" )
print( "%d %d %d %d" % (1, 2, 3, 4))
print( "1 ", end="")
print( "2 ", end="")
print( "3 ", end="")
print( "4" )